clang/lib/Driver/Driver.cpp

```diff
@@ -979,6 +1000,45 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
     return;
   }
 
+  //
+  // SYCL
+  //
+  // We need to generate a SYCL toolchain if the user specified -fsycl.
```

Comment:
I think this PR will benefit from updating SYCLSupport.rst document. I can help with that if need be. Thanks